samples/core/WPF/GetStartedWPF/GetStartedWPF/MainWindow.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<Window x:Class="GetStartedWPF.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:GetStartedWPF"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800" Loaded="Window_Loaded">
+    <Window.Resources>
+        <CollectionViewSource x:Key="categoryViewSource"/>
+        <CollectionViewSource x:Key="categoryProductsViewSource" 
+                              Source="{Binding Products, Source={StaticResource categoryViewSource}}"/>
+    </Window.Resources>
+    <Grid DataContext="{StaticResource categoryViewSource}">
```

Comment:
Seems a little unorthodox to specify DataContext. I'd just use `ItemsSource="{Binding Source={StaticResource categoryViewSource}}"` in the DataGrid below.